runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -5257,6 +5257,7 @@ class INTERPRETER_CLASS
 				/* Reset the virtual thread's notified field before releasing Object monitor. */
 				J9VMJAVALANGVIRTUALTHREAD_SET_NOTIFIED(_currentThread, _currentThread->threadObject, JNI_FALSE);
 				/* Try to yield the virtual thread if it will be blocked. */
+				J9VMJAVALANGVIRTUALTHREAD_SET_STATE(_currentThread, _currentThread->threadObject, newState);
```

Comment:
please add a comment here to mention that this is to prevent `Object.notify` occuring between prepareVirtualThreadForUnmount and yieldPinnedContinuation to be ignored by unblocker.

Reply:
It is to prevent object.notify from being ignored by unblocker inbetween the two function call, not to prevent the call itself

Reply:
can you also update the PR description please